web/src/Web.App/ViewComponents/AnalyticsViewComponent.cs

```diff
@@ -14,7 +14,12 @@ public IViewComponentResult Invoke()
         }
 
         var cookiePolicy = HttpContext.Request.Cookies[Constants.CookieSettingsName];
-        var vm = new AnalyticsViewModel(instrumentationKey, cookiePolicy != "disabled");
+        var vm = new AnalyticsViewModel(instrumentationKey, cookiePolicy switch
+        {
+            "enabled" => true,
+            "disabled" => false,
+            _ => null
```

Comment:
Should this be false, rather than null? So that it's always disabled if it falls through.

Reply:
On the other side I do a check for `== true` which has the same effect. Happy to switch over if you prefer

Reply:
I can't remember off the top of my head - doesn't null or undefined with double equals return true?

Reply:
Yes, in JavaScript if you just perform a reference check. I perform a [value check in the Razor view](https://github.com/DFE-Digital/education-benchmarking-and-insights/pull/1119/files/e7f5167371b43b6697ffea48f2e89a84eaaf8977#diff-395a70c272e43a5b541cc6d687b29b76f9ef1857716f368db8f0446225d4c4e5R24).

Reply:
If it's not clear I'll just swap over

Reply:
Reverted to avoid confusion